lib/SnippetsTestMessageProvider.jsm

```diff
@@ -402,6 +415,27 @@ const MESSAGES = () => [
       block_button_text: "Block",
     },
   },
+  {
+    id: "SPECIAL_SNIPPET_MONITOR",
+    template: "simple_below_search_snippet",
+    content: {
+      icon: TEST_ICON,
+      title: "See if you've been part of an online data breach.",
+      text: "Firefox Monitor tells you what hackers already know about you.",
+      button_label: "Get monitor",
+      button_action: "ENABLE_FIREFOX_MONITOR",
+      button_action_args: {
+        url:
+          "https://monitor.firefox.com/oauth/init?utm_source=snippets&utm_campaign=monitor-snippet-test&form_type=email&entrypoint=newtab",
```

Comment:
I see both url and flowRequestParams. Can you drop a line about when to use what? @k88hudson

Reply:
Yep, I will add some docs. Essentially the URL is everything snippets would be able to provide, the `flowParams` is a few properties needed to pass to a Firefox metrics endpoint (defined via a pref) in order to get the right `flowId`, `flowBeginTime` and `deviceId` back.